src/components/Pressable/PressableWithDelayToggle.js

```diff
@@ -13,7 +13,7 @@ import useThrottledButtonState from '../../hooks/useThrottledButtonState';
 
 const propTypes = {
     /** Ref passed to the component by React.forwardRef (do not pass from parent) */
-    innerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]).isRequired,
+    innerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]),
```

Comment:
@rushatgabhane Yes sure, the CopyTextToClipboard component uses the PressableWithDelayToggle Component as a child. And here, no ref passed to the PressableWithDelayToggle from CopyTextToClipboard so that the child component is getting the null value for innerRef And `React.forwardRef` will pass `null` value to `ref`
https://github.com/Expensify/App/blob/6a383413758030d687a728f66b19b00eadab7f3c/src/components/CopyTextToClipboard.js#L23-L32

Reply:
Thanks!